source/information-types/concept.txt

```diff
@@ -0,0 +1,62 @@
+======================
+Concept Page Prototype
+======================
+
+A concept page helps a user learn about a concept or new feature. 
+It is intended to answer the questions "what is this?", "why do I care", and 
+"how does this work?". The audience for a concept page is users who have never
+used a particular feature or command. 
+
+The title is typically a noun or noun phrase. 
+
+A concept page begins with a brief introduction summarizing its contents, often 
+called a short description. The short description explains a little about the 
+concept and why the audience should care about it.
+
+If it's appropriate, you may include a code-block with a copyable
+prototype to familiarize users with command syntax:
+
```

Comment:
Consider adding a heading before and a "Link to the reference page" after the code block.

Reply:
@davemungo thanks for the comment. I updated the layout here to include a heading and dive into a bit more detail on linking strategy.